htdocs/includes/odtphp/odf.php

```diff
@@ -304,7 +304,7 @@ private function _replaceHtmlWithOdtTag($tags, &$customStyles, &$fontDeclaration
 							}
 							if (strlen($odtStyles) > 0) {
 								// Generate a unique id for the style (using microtime and random because some CPUs are really fast...)
-								$key = floatval(str_replace('.', '', microtime(true))) + rand(0, 10);
+								$key = str_replace('.', '', microtime(true)) . uniqid() . rand(0, 10); // fixed by tronx2100
```

Comment:
It seems uniqid() is not unique when calling several time fast (see https://www.php.net/manual/fr/function.uniqid.php). 
May be you can call 
`uniqid(mt_rand(), false)`
we do this in a lot of other part of dolibarr code...